classes/fields/datetime.php

```diff
@@ -307,6 +307,12 @@ public function input( $name, $value = null, $options = null, $pod = null, $id =
 	 * {@inheritdoc}
 	 */
 	public function validate( $value, $name = null, $options = null, $fields = null, $pod = null, $id = null, $params = null ) {
+		$return = parent::validate( $value, $name, $options, $fields, $pod, $id, $params );
+
+		$errors = array();
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: f5578e99b777625b351dc771a41ffdfc</sub>